examples/get-started/pure-js/maplibre-globe/app.js

```diff
@@ -0,0 +1,62 @@
+// deck.gl
+// SPDX-License-Identifier: MIT
+// Copyright (c) vis.gl contributors
+
+import {MapboxOverlay as DeckOverlay} from '@deck.gl/mapbox';
```

Comment:
Why rename/alias things? If we are not happy with the name, rename it at export? If trying to make code clearer a longer name might be needed? Any name that mentions either just Deck or Mapbox is going to be confusing